src/main/webapp/WEB-INF/tags/instructor/home/sort.tag

```diff
@@ -28,5 +28,9 @@
                 </label>
             </div>
         </div>
+        <div class="col-md-3">
+            <a class="btn btn-primary btn-md pull-right" href="${data.instructorCourseLink}" 
+                id="addNewCourse">Add New Course</a>
+        </div>
```

Comment:
Kind of irrelevant to put the add new course button inside a sort tag, don't you think? Or is this because of artistic purpose?